website/src/pages/index.js

```diff
@@ -106,6 +107,11 @@ function Home() {
               Get Started
             </Link>
           </div>
+
+        
+          <GitHubButton href="https://github.com/h2oai/wave" data-color-scheme="no-preference: light; light: light; dark: dark;" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star h2oai/wave on GitHub">Star</GitHubButton>
+
+
```

Comment:
Formatting:

* remove extra blank lines
* line is quite long, separate attributes with newlines